### ISSUE TITLE
Locking Focal Length When Capturing with Canon 5D MKIII

### ISSUE THREAD

**Issue by AuroraRyan0301** [YilingQiao/DMRF#4]
Hi, @gaoalexander,

I noticed that your dataset lacks EXIF information, such as ISO, focal length, and aperture values. I wonder if you locked the focus while capturing the images. Even after switching from AF to MF mode( a switch on lens), the camera still slightly adjusts the focus when the shutter button is half-pressed. It causes me some trouble when I try to make a dataset with such photos. 
I read the manual of Canon 5D MKIII and didn't find a way to lock the focus. I would like to know how you addressed this issue and managed to capture images with a locked focus.

Could you please provide some insights on this?

Thank you!

**Comment by gaoalexander** [comment]
Hi @AuroraRyan0301, it's surprising to me that the focus is still automatically adjusted, even though you have set the lens to MF.  I will say that you should put the camera body itself (not just the lens) in fully Manual mode, which will allow you to set your ISO, exposure, etc. without them changing at all between shots.

What kind of lens are you using btw?  Happy to help troubleshoot more if you're still having trouble after switching to full Manual mode.

**Comment by AuroraRyan0301** [comment]
Thank you @gaoalexander  for your response. We used the original lens EF24-105L with a Canon 5D Mark III and moved the mode dial to "M", which I assume is what you referred to as full Manual mode. We discovered that there was a slight adjustment (4 - 8 mm) in the focal length when half-pressing the shutter button, even in the M mode. Through experimentation, we managed to solve this issue using a somewhat less elegant method – we taped the adjustment ring which is used to adjust the fov.

Perhaps there are other settings that could fundamentally address this issue. Fortunately, we were able to resolve it for our data shooting using this method.

Thank you for your response again.
